client/my-sites/themes/controller.js

```diff
@@ -68,6 +69,20 @@ function makeElement( ThemesComponent, Head, store, props ) {
 	);
 };
 
+// Where do we put this? It's client/server-agnostic, so not in client/controller,
+// which requires ReactDom... Maybe in lib/react-helpers?
+export function makeLoggedOutLayout( context, next ) {
+	const { store, primary, secondary, tertiary } = context;
+	context.layout = (
+		<ReduxProvider store={ store }>
+			<LayoutLoggedOut primary={ primary }
+				secondary={ secondary }
+				tertiary={ tertiary } />
+		</ReduxProvider>
+	);
+	next();
+};
```

Comment:
Mhm, been considering that, too. Right now I'd rather not delete them, since the middleware kind of guarantees that they shouldn't do any harm, so seeing them deleted might cause even more confusion (what would happen if they weren't deleted)...